airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceMetadataQuerier.kt

```diff
@@ -37,7 +37,6 @@ class MysqlSourceMetadataQuerier(
                     Pair("show variables where Variable_name = 'log_bin'", "ON"),
                     Pair("show variables where Variable_name = 'binlog_format'", "ROW"),
                     Pair("show variables where Variable_name = 'binlog_row_image'", "FULL"),
-                    Pair("show variables where Variable_name = 'gtid_mode'", "ON"),
```

Comment:
@xiaohansong Why did we add this check to begin with?
debezium [docs](https://debezium.io/documentation/reference/stable/connectors/mysql.html#enable-mysql-gtids) states it is "not required".